### ISSUE TITLE
restore.sh does not honor MYSQL_DB

### ISSUE THREAD

**Issue by bcardiff** [tutumcloud/mysql-backup#5]
if `MYSQL_DB` is set, the restore.sh generated does not work.
No database is created.
And the .sql script is executed without a database been selected.

**Comment by fermayo** [comment]
@bcardiff can you add the `USE <dbname>;` command at the beginning of the SQL script?

**Comment by bcardiff** [comment]
Yes @fermayo I can. But when using `MYSQL_DB` in this docker image, the scripts are generated without a `USE <dbname>` and so the restore cycle is not applicable in a straight way.

If `MYSQL_DB` is set, restore.sh should perform a `CREATE DATABASE IF NOT EXISTS <dbname>;` and also use `./mysql` with `MYSQL_DB` as a final argument IMO.
